### PR TITLE
Revert "Revert "split verify out of unit/integration suite""

### PR DESCRIPTION
**Pull request by mikedanese**
Reverts kubernetes/test-infra#163

### CODE REVIEW

**Comment by spxtr** [comment]
The github revert button is annoying and creates a branch in the main repo. I think we just need to fix the diff script to not break when we're not merging into master...

File: jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml [hunk 1]
```diff
@@ -194,9 +194,7 @@
             colormap: xterm
     builders:
         - shell: 'JENKINS_BUILD_STARTED=true bash <(curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh")'
-        - shell: |
-            export KUBE_VERIFY_GIT_BRANCH="${ghprbTargetBranch}"
-            ./hack/jenkins/gotest-dockerized.sh
+        - shell: '{cmd}'
     publishers:
         - xunit:
```
Comment (ixdy) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: the verify job isn't creating junit here, either.

does the xunit publisher care?

Comment (mikedanese) [inline jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml]: It is until https://github.com/kubernetes/kubernetes/pull/27289 is merged, so we won't know.

**Comment by wojtek-t** [comment]
@mikedanese - I took a look into "verify" jenkins job and it is running unit & integration tests too. Is that a known issue?

**Comment by mikedanese** [comment]
We have a plan to phase in this change to not be disruptive. As part of that plan, there is a brief period of double testing, which we are currently in. That will change once https://github.com/kubernetes/kubernetes/pull/27289 is merged (probably today).

**Comment by wojtek-t** [comment]
@mikedanese - sounds like a great plan; thanks for explanation!

**Pull request merged**